gallery/src/styles/NxClickable/NxClickableExample.scss

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @import '~@sonatype/react-shared-components/scss-shared/nx-text-helpers';
+
+.nx-truncate-example1 {
```

Comment:
This file seems wrong and also like it shouldn't exist

Reply:
> This file seems wrong and also like it shouldn't exist in this PR

Yeah that should have been deleted: https://github.com/sonatype/sonatype-react-shared-components/pull/67/commits/095a7c8802e9560e4579013ed4326161b2083901